<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qimageshowconfig</class>
 <widget class="QDialog" name="qimageshowconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>158</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">qimageshowconfig</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Aspect Ratio:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Scale Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sodoku Navigation Key:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="aspectRatiocomboBox">
           <item>
            <property name="text">
             <string>Ignore Aspect Ratio</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Keep Aspect Ratio</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="scaleModecomboBox">
           <item>
            <property name="text">
             <string>Near</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bilinear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Area</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bicubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lanczos</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sodokuModecomboBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Num1~9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Arrow Key</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="squarecheckBox">
         <property name="text">
          <string>Square</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="squareratioxspinBox">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="squareratioyspinBox">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="crosscheckBox">
       <property name="text">
        <string>Cross</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="savepushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
